changelog.dd

```diff
@@ -175,6 +176,10 @@ $(BUGSTITLE Compiler Changes,
         }
         ---
     )
+
+    $(LI $(LNAME2 dash_safe, Add -transition=safe switch.)
+	$(P Enables enhanced @safe checking, which will break some existing code.)
```

Comment:
Could you please enumerate these enhancements? This is the change log after all.